operators/elasticsearch-eck-operator-certified/2.1.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,8 @@
+annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: stable
+  operators.operatorframework.io.bundle.channels.v1: stable
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: elasticsearch-eck-operator-certified
+  com.redhat.openshift.versions: "v4.6-v4.10"
```

Comment:
This should be just "v.4.6" not a closed version range

Reply:
fixed.  Thanks @pebrc !